lfs/gitscanner_refs.go

```diff
@@ -91,6 +92,13 @@ func scanRefsToChan(scanner *GitScanner, pointerCb GitScannerFoundPointer, refLe
 	return nil
 }
 
+// scanLeftRightToChan takes a ref and returns a channel of WrappedPointer objects
+// for all Git LFS pointers it finds for that ref.
+// Reports unique oids once only, not multiple times if >1 file uses the same content
+func scanLeftRightToChan(scanner *GitScanner, pointerCb GitScannerFoundPointer, refLeft, refRight string, opt *ScanRefsOptions) error {
+	return scanRefsToChan(scanner, pointerCb, []string{refLeft, refRight}, nil, opt)
```

Comment:
Hm, should this be:

```go
return scanRefsToChan(..., []string{refLeft}, []string{refRight}, nil, opt)
```

instead? I think that they should be split, because the right is a boundary of everything we don't want in the range.

Reply:
I think this is correct as-is. My gut reaction was the same, but 

```
[]string{refLeft, refRight}, nil
```
was being used internally by `scanRefsToChan()` before I changed its functionality here. Additionally, changing it to 

```
[]string{refLeft}, []string{refRight},
```
causes tests that leverage this function to fail. I'm still doing some investigation as to why and will come back with an update when I better understand the rev scanner.

Reply:
Ah, you're totally right. I just checked the code and the way that it is in your diff is 👍, my comment is incorrect.

Reply:
@ttaylorr I've gone ahead and addressed your feedback! I've also added the `--remote` flag and docs